{"content": {"suites": {"0": {"status": {"total_pass": 9, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}, "tests": {"0": {"status": "PASS", "message": "", "test_name": "test_reverse", "rerun": "0"}, "1": {"status": "PASS", "message": "", "test_name": "test_reverse_argument", "rerun": "0"}, "2": {"status": "PASS", "message": "", "test_name": "test_complement", "rerun": "0"}, "3": {"status": "PASS", "message": "", "test_name": "test_complement_argument", "rerun": "0"}, "4": {"status": "PASS", "message": "", "test_name": "test_validation", "rerun": "0"}, "5": {"status": "PASS", "message": "", "test_name": "test_type_validation", "rerun": "0"}, "6": {"status": "PASS", "message": "", "test_name": "test_nucleotipe_validation", "rerun": "0"}, "7": {"status": "PASS", "message": "", "test_name": "test_reverse_complement", "rerun": "0"}, "8": {"status": "PASS", "message": "", "test_name": "test_reverse_complement_argument", "rerun": "0"}, "9": {"status": "FAIL", "message": "   TypeError: argument of type 'Read_file' is not iterable\n", "test_name": "test_load_data", "rerun": "0"}}, "suite_name": "test/test.py"}}}, "date": "September 24, 2021", "start_time": 1632482209.972809, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "9", "fail": "1", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "10"}